pennylane/ops/op_math/symbolicop.py

```diff
@@ -106,14 +106,6 @@ def _wires(self, new_wires):
     def num_wires(self):
         return len(self.wires)
 
-    @property
-    def batch_size(self):
-        return self.base.batch_size
-
-    @property
-    def ndim_params(self):
-        return self.base.ndim_params
-
```

Comment:
Why remove these? I would imagine we can just override them for the exceptions?

Reply:
This allows it to work.  Not 100% sure how `op._check_batching()` works, so I can't exactly say why.